configs/text-gen-service/aws.yaml

```diff
@@ -0,0 +1,8 @@
+head_node_type:
+  name: head_node_type
+  instance_type: g4dn.xlarge
```

Comment:
Is this intentionally using a GPU head node?

Reply:
not really. will fix it